closed/src/java.base/share/classes/openj9/internal/criu/NativePRNG.java

```diff
@@ -245,5 +280,92 @@ public OutputStream run() {
                 }
             }
         }
+
+        // ensure that there is at least one valid byte in the buffer
+        // if not, read new bytes
+        private void ensureBufferValid() throws IOException {
+            long time = System.currentTimeMillis();
+            int new_buffer_size = 0;
+
+            // Check if buffer has bytes available that are not too old
```

Comment:
Please either finish each sentence with a period, or don't capitalize the first word of a sentence fragment.

Reply:
updated